libpod/image/pull_test.go

```diff
@@ -150,33 +150,25 @@ func TestPullGoalFromImageReference(t *testing.T) {
 		{ // RepoTags is empty
 			"docker-archive:testdata/docker-unnamed.tar.xz",
 			[]expected{{"@ec9293436c2e66da44edb9efb8d41f6b13baf62283ebe846468bc992d76d7951", "@ec9293436c2e66da44edb9efb8d41f6b13baf62283ebe846468bc992d76d7951"}},
-			false,
+			true,
 		},
 		{ // RepoTags is a [docker.io/library/]name:latest, normalized to the short format.
-			"docker-archive:testdata/docker-name-only.tar.xz",
-			[]expected{{"localhost/pretty-empty:latest", "localhost/pretty-empty:latest"}},
+			"docker-archive:testdata/docker-name-only.tar.xz:pretty-empty:latest",
+			[]expected{{"docker.io/library/pretty-empty:latest", "docker.io/library/pretty-empty:latest"}},
 			true,
 		},
 		{ // RepoTags is a registry/name:latest
-			"docker-archive:testdata/docker-registry-name.tar.xz",
+			"docker-archive:testdata/docker-registry-name.tar.xz:example.com/empty:latest",
 			[]expected{{"example.com/empty:latest", "example.com/empty:latest"}},
 			true,
 		},
 		{ // RepoTags has multiple items for a single image
-			"docker-archive:testdata/docker-two-names.tar.xz",
+			"docker-archive:testdata/docker-two-names.tar.xz:example.com/empty:latest",
 			[]expected{
-				{"localhost/pretty-empty:latest", "localhost/pretty-empty:latest"},
 				{"example.com/empty:latest", "example.com/empty:latest"},
 			},
 			true,
 		},
-		{ // FIXME: Two images in a single archive - only the "first" one (whichever it is) is returned
```

Comment:
I’d expect a test to remain with `len(expected) == 0` to test that this is rejected.